bin/pygrb/pycbc_pygrb_plot_snr_timeseries

```diff
@@ -203,7 +202,8 @@ xlims = [start, end]
 if opts.x_lims:
     xlims = opts.x_lims
     xlims = map(float, xlims.split(','))
+
 plu.pygrb_plotter([trig_data_time, trig_data_snr],
                   [inj_data_time, inj_data_snr],
                   "Time since %.3f (s)" % (central_time), y_label,
-                  opts, cmd=' '.join(sys.argv))
+                  opts, cmd=' '.join(sys.argv), fig=plt.figure())
```

Comment:
Is this change (and the corresponding one to `pygrb_plotter()` below) really needed? I do not see a functional change.

Reply:
I added it because I thought it'd be nice to be able to pass an existing pyplot.figure object to multiple instances of  pygrb_plotter to plot different datasets when debugging.  For example, the [plot](https://private-user-images.githubusercontent.com/92443572/345035725-d7872fd6-7cf9-4245-88f3-f4e3fc2b35e1.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjEzOTgwMzAsIm5iZiI6MTcyMTM5NzczMCwicGF0aCI6Ii85MjQ0MzU3Mi8zNDUwMzU3MjUtZDc4NzJmZDYtN2NmOS00MjQ1LTg4ZjMtZjRlM2ZjMmIzNWUxLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA3MTklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNzE5VDE0MDIxMFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWZhZTI3Mzc1MTQzNDBhZDFlODVlYTZmZTkxNzAwNzIxOWY1OGM5MjRlNzI5MjNlY2E0NTVhMjhkNjQ5NDhhOTAmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.F2RS-9lSLWCvO9FzIoNMson6FobCPs1QewvscsdyR-0), was easily produced with such feature. 

In any case, I'm happy to remove it if you think such modification is not needed. Let me know.